pkg/virt-launcher/notify-client/client.go

```diff
@@ -219,3 +209,28 @@ func StartNotifier(virtShareDir string, domainConn cli.Connection, deleteNotific
 	log.Log.Infof("Registered libvirt event notify callback")
 	return nil
 }
+
+func (c *NotifyClient) SendK8sEvent(vmi *v1.VirtualMachineInstance, severity string, reason string, message string) error {
+
+	reply := &notifyserver.Reply{}
+
+	event := k8sv1.Event{
+		InvolvedObject: k8sv1.ObjectReference{
```

Comment:
Can you fill in the whole reference? At least also the `UID`. Otherwise we can't distinguish to which object instance the message belongs.

Reply:
done